src/polyfem/solver/line_search/LineSearch.tpp

```diff
@@ -6,6 +6,8 @@
 #include "CppOptArmijoLineSearch.hpp"
 #include "MoreThuenteLineSearch.hpp"
 
+#include <fstream>
```

Comment:
remove?

Reply:
No, this is needed for `save_sampled_values()`. It was missing and causing compiler errors if headers were included in a different order.